pkg/auth/providers/local/local_auth_provider.go

```diff
@@ -420,11 +424,24 @@ func minOf(length int, defaultLen int) int {
 	return defaultLen
 }
 
-func indexField(field string, maxindex int) []string {
+func indexField(field string, maxIndex int) []string {
 	var fieldIndexes []string
-	for i := 2; i <= maxindex; i++ {
-		fieldIndexes = append(fieldIndexes, field[0:i])
+	for i := 2; i <= maxIndex; i++ {
+		fieldIndexes = append(fieldIndexes, strings.ToLower(
+			string([]rune(simplifyString(field))[0:i])))
 	}
+
+	splitToLower := func(s string, limit int) []string {
+		var lowers []string
+		for _, v := range strings.Fields(s) {
+			lowers = append(lowers, strings.ToLower(v)[:minOf(limit, len(v))])
+		}
+
+		return lowers
+	}
+
+	fieldIndexes = append(fieldIndexes, splitToLower(field, maxIndex)...)
```

Comment:
`fieldIndexes`  has duplicated items as `fieldIndexes` and `splitToLower(field, maxIndex) `will both contain the first word. That's not a problem in `userSearchIndexer` because we are using a set. However, `groupSearchIndexer` will contain duplicated items.

Reply:
Ahhh...good point, lemme fix that

Reply:
I removed the duplication, it only adds those fields for the user indexer.